src/hotspot/share/oops/generateOopMap.cpp

```diff
@@ -2057,7 +2057,7 @@ void GenerateOopMap::print_time() {
   tty->print_cr ("---------------------------");
   tty->print_cr ("  Total : %3.3f sec.", GenerateOopMap::_total_oopmap_time.seconds());
   tty->print_cr ("  (%3.0f bytecodes per sec) ",
-  (float)GenerateOopMap::_total_byte_count / GenerateOopMap::_total_oopmap_time.seconds());
+  (double)GenerateOopMap::_total_byte_count / GenerateOopMap::_total_oopmap_time.seconds());
```

Comment:
I still don't see a need for the double cast here - the normal numeric promotion rules will make the expression a double. What error is reported for this code?

Reply:
/hotspot/share/oops/generateOopMap.cpp: In static member function 'static void GenerateOopMap::print_time()':
/hotspot/share/oops/generateOopMap.cpp:2060:19: warning: conversion from 'uint64_t' {aka 'long unsigned int'} to 'double' may change value [-W
conversion]
 2060 |   GenerateOopMap::_total_byte_count / GenerateOopMap::_total_oopmap_time.seconds());
      |   ~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~

Reply:
Ah now I see. Thanks.